recipes/cfitsio/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "cfitsio" %}
+{% set version = "3.410" %}
+{% set nodotversion = "3410" %}
+{% set sha256 = "a556ac7ea1965545dcb4d41cfef8e4915eeb8c0faa1b52f7ff70870f8bb5734c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}{{ nodotversion }}.tar.gz
+  url: ftp://heasarc.gsfc.nasa.gov/software/fitsio/c/{{ name }}{{ nodotversion }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libcfitsio.a
+
+about:
+  home: http://heasarc.gsfc.nasa.gov/fitsio/fitsio.html
+  license: Unspecified
```

Comment:
Ah, it turns out there is something to be referenced here: [see this page](https://heasarc.gsfc.nasa.gov/docs/software/fitsio/c/f_user/node9.html). It looks like an MIT-ish license but the disclaimer does point out that it's not under copyright. Suggestions as to how to describe the license in the package?